### PR TITLE
New package: bro-2.5.4

### PR DESCRIPTION
**Pull request by abenson**

### CODE REVIEW

**Comment by maxice8** [comment]
musl-fts-devel isn't available on non -musl arches

**Comment by abenson** [comment]
I'll fix it, I still need to get it to work at all on musl.

    src/tcmalloc.cc:332] Attempt to free invalid pointer 0x7ffff6041fe0

**Comment by abenson** [comment]
Didn't meant to commit this like that.

File: srcpkgs/bro/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+maintainer="Andrew Benson <abenson+void@gmail.com>"
+license="BSD-3-Clause"
+homepage="https://www.bro.org"
+distfiles="https://www.bro.org/downloads/bro-${version}.tar.gz"
+checksum=80daea433fa654f2602cf67b19b9121ff6ad57761bad73cc29020c4f490c5f1f
+
+case "$XBPS_TARGET_MACHINE" in
+	*-musl)
+		makedepends+=" musl-fts-devel"
+		sed -i '/set(broxygen/i link_libraries("-lfts")' src/broxygen/CMakeLists.txt
```
Comment (maxice8) [inline srcpkgs/bro/template]: This is run when the template is sourced, put it inside pre_configure()

**Comment by maxice8** [comment]
bro-config should be in -devel package

**Comment by abenson** [comment]
That doesn't make sense since it doesn't have anything to do with developing for bro

**Comment by maxice8** [comment]
> That doesn't make sense since it doesn't have anything to do with developing for bro

i see, i assumed it was like the other -config binaries.

**Comment by abenson** [comment]
No, it's for runtime stuff, so things like bro-pkg can find the location of installation

**Comment by maxice8** [comment]
On Sun, Jun 24, 2018 at 08:15:18AM -0700, Andrew Benson wrote:
> No, it's for runtime stuff, so things like bro-pkg can find the location of installation
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/303#issuecomment-399764021

I assumed that after i saw that bro doesn't have -devel stuff

**Pull request merged**